languages/javascript/exercises/concept/array-analysis/.meta/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "contributors": [
+    {
+      "github_username": "peterchu999",
+      "exercism_username": "peterchu999"
+    }
+  ],
+  "authors": [
+    {
+      "github_username": "peterchu999",
+      "exercism_username": "peterchu999"
+    }
+  ]
+}
```

Comment:
Can you add me (`SleeplessByte`) to the contributors?

Reply:
sure